### PR TITLE
Add ibis-vega-transform

### PR DESCRIPTION
**Pull request by xmnlab**
This PR adds a post-install step that:

- installs ibis-vega-transform, and
- installs jupyterlab extensions ibis-vega-transform and @pyviz/jupyterlab_pyviz

### CODE REVIEW

**Comment by xmnlab** [comment]
This PR is ready for the review. Thanks! :)

File: constructor/post-install.sh [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+export PATH=$PREFIX/bin:$PATH
+
+pip install ibis-vega-transform
```
Comment (scopatz) [inline constructor/post-install.sh:5]: Instead you might want to do `$PREFIX/bin/python -m pip`

File: constructor/post-install.sh [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+
+jupyter labextension install ibis-vega-transform
+jupyter labextension install @pyviz/jupyterlab_pyviz
```
Comment (scopatz) [inline constructor/post-install.sh:8]: Similarly, you might want to do `$PREFIX/bin/jupyter`

File: constructor/pkg/construct.yaml [hunk 3]
```diff
@@ -21,8 +21,11 @@ specs:
   - jupyter-jaeger
   - pandas <0.26
   - pymapd
+  - pyarrow 0.15.0
```
Comment (scopatz) [inline constructor/pkg/construct.yaml:24]: Can this be `>=0.15.0`

File: constructor/pkg/construct.yaml [hunk 4]
```diff
@@ -21,8 +21,11 @@ specs:
...
   - python >=3.7
   - ibis-framework
   - jupyterlab >=2.2
   # needed for pkg building
   - conda
+
+post_install: ../post-install.sh  # [osx]
```
Comment (scopatz) [inline constructor/pkg/construct.yaml:31]: IIRC correctly post_install scripts are not run for the pkg installer due to a bug in constructor.  You should check this.

Comment (xmnlab) [reply constructor/pkg/construct.yaml:31]: @scopatz as pkg is not priority for now, could I work on that as a follow-up task?
locally the building is taking a lot of time ... 
I can remove the package pkg from the release if it is not working (the priority right now is the sh)
I can also check (for a follow-up task) if I can check the pkg inside the CI

how does it sound?

Comment (scopatz) [reply constructor/pkg/construct.yaml:31]: Yeah that's fine

**Pull request merged**